languages/wikipron/src/frequencies/grab_wortschatz_data.py

```diff
@@ -51,6 +48,9 @@ def unpack():
 
 
 def main():
+    with open("wortschatz_languages.json", "r") as langs:
```

Comment:
make this string a global variable at the top?

Reply:
Or perhaps I should add it to the other path constants in `src/codes.py`?